org.lamport.tla.toolbox.yml

```diff
@@ -0,0 +1,55 @@
+app-id: org.lamport.tla.toolbox
+runtime: org.freedesktop.Platform
+runtime-version: "19.08"
+sdk: org.freedesktop.Sdk
+command: tlatoolbox
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

Comment:
Does it actually work under Wayland or just uses xwayland? If the latter, please drop this line.

Reply:
I've just tested this build on Wayland and it works.

Reply:
The question here is if it works natively or fall backs to xwayland.

Reply:
Ok, I rebuilt without wayland socket and it works still. So I guess this arg is not really used.

Reply:
it's more the other way around, if you start it with --nosocket=x11 under wayland (and --socket=wayland, obviously) does it still work fine?

Reply:
(you can override permissions on the command line for a specific launch, in case you didn't know so `flatpak run --nosocket=x11 org.lamport.tla.toolbox`

Reply:
> it's more the other way around, if you start it with --nosocket=x11 under wayland (and --socket=wayland, obviously) does it still work fine?

Just checked it: no, it doesn't work without x11 socket under wayland.

Reply:
Let's drop `--socket=wayland` then. Ready to merge when it's done.

Reply:
Socket removed. Thank to both of you for your advice :)